libs/sr/ui/src/Character/CharacterCard.tsx

```diff
@@ -0,0 +1,21 @@
+import { CardThemed } from '@genshin-optimizer/common/ui'
+import type { ICachedSroCharacter } from '@genshin-optimizer/sr/db'
+import { CardContent, Stack } from '@mui/material'
+
+export function CharacterCard({
+  character,
+}: {
+  character: ICachedSroCharacter
+}) {
+  return (
+    <Stack>
+      <CardThemed>
+        <CardContent>
+          <div>key: {character.key}</div>
+          <div>eidolon: {character.eidolon}</div>
+          <div>level: {character.level}</div>
```

Comment:
Need to add stat calcs here. I will come back with more guidance shortly (maybe 1d)

Reply:
Add these lines before the return
```
  const calc = srCalculatorWithEntries(
    withMember('member0', ...charData(character))
  )
  const member0 = convert(selfTag, { member: 'member0', et: 'self' })
```

then you can do
`calc.compute(member0.final.<stat>).val` such as `calc.compute(member0.final.atk).val`

Can probably show all of these to start, with appropriate labels

![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/6a0b3b3b-af01-48f6-b7fd-de6a8ae65c2c)

Reply:
Done